roles/ngi-rnaseq/tasks/dependencies.yml

```diff
@@ -44,7 +47,25 @@
   - { envvar: "export NXF_VER={{ nextflow_ver.stdout }}"}
   - { envvar: "export NXF_HOME={{ nextflow_dest }}/workfiles"}
   - { envvar: "export NXF_TEMP=${SNIC_TMP:-{{ nextflow_dest }}/workfiles}"}
-  - { envvar: "export NXF_WORK=${SNIC_TMP:-{{ nextflow_dest }}/workfiles}"}
+  - { envvar: "export NXF_WORK={{ nextflow_dest }}/workfiles"}
   - { envvar: "export NXF_LAUNCHBASE=${SNIC_TMP:-{{ nextflow_dest }}/workfiles}"}
   - { envvar: "export NXF_LAUNCHER=${SNIC_TMP:-{{ nextflow_dest }}/workfiles}"}
 
+- name: Copy R dependencies script
+  copy:
+    src: "files/install_R_dependencies.r"
+    dest: "{{ nextflow_dest }}/r_packages"
+ 
+- name: Load R module
+  shell: "module load R"
```

Comment:
You know that all these `shell` and `command` calls live in their own separate environments right? They will not know about each other.